.gitignore

```diff
@@ -6,3 +6,6 @@ render.ppm
 doxygen/html
 examples/CubeOfSpheres
 examples/ObjFile
+disasm/*.lst
```

Comment:
This wasn't technically suppose to make it in this commit. Sorry about that! This directory was moved into my development repo and was for examining disassembly of some methods and viewing optimization diagnostics.